src/legacy/core_plugins/kibana/public/management/sections/index_patterns/create_index_pattern_wizard/components/step_time_field/step_time_field.js

```diff
@@ -50,8 +50,6 @@ export class StepTimeFieldComponent extends Component {
   constructor(props) {
     super(props);
 
-    const { getIndexPatternType, getIndexPatternName } = props.indexPatternCreationType;
```

Comment:
Destructuring here made arrow functions work while non-arrow functions break. I believe it's a bit unintuitive. So, I've removed this destructuring and called functions directly below.